etl/PAIMS_PIMS/PAIMS_PIMS/MOTI_LEGACY_PAIMS_PIMS_DATA_MIGRATION.dtproj

```diff
@@ -26,295 +26,330 @@
           <SSIS:Property SSIS:Name="CreatorComputerName">XERCES</SSIS:Property>
           <SSIS:Property SSIS:Name="Description">
           </SSIS:Property>
-          <SSIS:Property SSIS:Name="PasswordVerifier" SSIS:Sensitive="1">AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAAsHzQ0qRXJ0u1ypPOd2k6DQAAAAACAAAAAAAQZgAAAAEAACAAAACssOzFFvXx2BLTpmFeerTHjkCI195WlOKYYPHy+t8dXgAAAAAOgAAAAAIAACAAAAAKme8tj97kqZaXawfcWFnw7svfB9oShIHgSJR1bc4rRpAAAAB5Ac4MLDyeXnsqSjjwyaAN05urkun5+oSTafMkFtldJEaqhdNIaNH8G6BKw0QEOxMVCzoCyM8IlpfZtLwaWheXPJMvJBOW/PfpfYztk6nd/vQBCfuY30Qmk6xUgLYAdZvaLZqNlcm64OYgB7vLHH9yDAKYewnqjsIsOegJNxUlOKmuxnptN2NiSEnHC59/0b1AAAAA4YN35k4si5fRjQPEAcKheDR0W7p2PmOLoItTv/SfgVh/Z0LqU2ZJdkE7O/V11WLT6WYFIZB2d8HMQPCRsSacBA==</SSIS:Property>
+          <SSIS:Property SSIS:Name="PasswordVerifier" SSIS:Sensitive="1">AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAA1mUTV2pQrESdnKDHC1rXUwAAAAACAAAAAAAQZgAAAAEAACAAAADdW8URgUeX7vRWWnKs3KfoFnfXeOikgi0H58J23AaGmgAAAAAOgAAAAAIAACAAAACN/5puLPaZ0VmjMpnSBx77pfwlR3SS4t6aBUKFxtla7pAAAABQHYgTF4kN846C0CV/qNrVP4v2qNvnL7QhJG/Q5TdAqA89iU8p8AKA/sbPAb2OL5dYxdqrCWIG2SdDgt/XUAOdGg4bfBYPR5MjJrA568TOnsGfmLWHdU1miFMw0Y+pa3qfFxBLLjkFfvzkY4CF4jWxFlhHCquucw7KBU1vpZ399aW3zGdxfh0edqTm5cjq7qtAAAAAGIBpi3QjamKZ+7QH9eaKReHUdPn3XAVLkM079NK5TIMfYAlI//HXuU/Vux3qsYYYsr0JD1iQj8EjKIB819bsxw==</SSIS:Property>
```

Comment:
This to me looks like the DontSaveSensitive flag hasn't been applied properly, or at least the sanitized dtproj hasn't been saved in git.

Reply:
it is set at the project level and at the package level in the UI? I wonder what is going on